openlibrary/plugins/upstream/utils.py

```diff
@@ -806,6 +806,7 @@ def get_language(lang_or_key: str) -> "None | Thing | Nothing":
         return lang_or_key
 
 
+@public
```

Comment:
We'll want to use the `convert_iso_to_marc` method below this one. That one supports all 1000+ languages in Open Library books.